x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/write_current_status_succeeded.ts

```diff
@@ -13,17 +13,32 @@ import { IRuleSavedAttributesSavedObjectAttributes } from '../rules/types';
 interface GetRuleStatusSavedObject {
   services: AlertServices;
   currentStatusSavedObject: SavedObject<IRuleSavedAttributesSavedObjectAttributes>;
+  lastLookBackDate: string | null | undefined;
+  bulkCreateTimes: string[] | null | undefined;
+  searchAfterTimes: string[] | null | undefined;
 }
 
 export const writeCurrentStatusSucceeded = async ({
   services,
   currentStatusSavedObject,
+  lastLookBackDate,
+  bulkCreateTimes,
+  searchAfterTimes,
 }: GetRuleStatusSavedObject): Promise<void> => {
   const sDate = new Date().toISOString();
   currentStatusSavedObject.attributes.status = 'succeeded';
   currentStatusSavedObject.attributes.statusDate = sDate;
   currentStatusSavedObject.attributes.lastSuccessAt = sDate;
   currentStatusSavedObject.attributes.lastSuccessMessage = 'succeeded';
+  if (lastLookBackDate) {
```

Comment:
nit -> we like do a condition like lastLookBackDate != null because it is easier to read and we know what you are talking about without going to the type